scripts/background.js

```diff
@@ -3,7 +3,7 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
   if (
     changeInfo.status === "complete" &&
     tabUrl &&
-    tabUrl.includes("someurl.com")
+    tabUrl.includes("hulu.com/hub/home")
   ) {
     chrome.scripting.insertCSS({
```

Comment:
It will inject global.css (which will override the existing CSS styles for the specified elements) only on the `hulu.com/hub/home` url.